tests/test_flows/test_flow.py

```diff
@@ -153,6 +153,23 @@ def test_publish_flow(self):
         flow.publish()
         self.assertIsInstance(flow.flow_id, int)
 
+    def test_semi_legal_flow(self):
```

Comment:
What exactly is tested here? OpenML should reject this flow, because it contains the bagging classifier twice.